### ISSUE TITLE
Can't navigate through code hints by arrows

### ISSUE THREAD

**Issue by widersky** [zed-industries/community#2036]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Just look at the short video below - I just can't navigate hints by arrow keys.

https://github.com/zed-industries/community/assets/45122321/df0d6be5-6176-41d1-ae25-587cea138af7

### Environment

Zed: v0.104.0 (preview)
OS: macOS 14.0.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
Woah, this is really strange.  I'm not sure what even might be causing this - I can't reproduce it at the moment.  Do you have any custom key bindings or settings? What happens if you (after backing up to another location) erase your custom bindings and settings, does the issue go away? Sometimes, we've ran into funky things happening where multiple settings cause some issue.  Let me know either way and we will dig further into whatever.

**Comment by widersky** [comment]
@JosephTLyons I just deleted all my settings from the `settings.json` file - I didn't set my own keyboard shortcuts. Anyway, I removed the settings from the `keymap.json` file to be sure, but it didn't help either. The problem still occurs. I have no problems navigating in this way either in VSCode or IntelliJ IDEA. It's also hard for me to say in which version of Zed the problem appeared - I'm testing every next version of Preview but I'm not working at Zed on a regular basis because I'm missing SCSS support 😅

**Comment by widersky** [comment]
@ConradIrwin @JosephTLyons Okay, now it works! I had a configuration in custom keyboard shortcuts (it's hard to say where it came from, maybe I changed something once, but there were 500+ lines so I would rather remember :D ). Either way - I replaced this configuration with the default one and it fixed the problem. Removing the configuration also works - perhaps this is due to the latest `0.104.4` update, because it did not help before. Let me close this issue :)
